app/(tabs)/guests.tsx

```diff
@@ -37,18 +38,28 @@ export default function App() {
     useCallback(() => {
       if (!isStale) return;
 
+      async function getUser() {
+        const response = await fetch('/api/user');
+
+        const body: UserResponseBodyGet = await response.json();
+
+        if ('error' in body) {
+          return <Redirect href="/(auth)" />;
```

Comment:
Is the `Redirect` component the correct pattern here? (instead of `router.push`)

If it is, let's make a comment about why (what the difference is, and if applicable also why `router.push` cannot be used here)

Reply:
No more usage of `<Redirect`